e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -200,23 +200,145 @@ test.describe('Example Imagery', () => {
+    await page.mouse.wheel(0, deltaYStep * 2);
+    const zoomedBoundingBox = await bgImageLocator.boundingBox();
+    const imageCenterX = zoomedBoundingBox.x + zoomedBoundingBox.width / 2;
+    const imageCenterY = zoomedBoundingBox.y + zoomedBoundingBox.height / 2;
+
+    // Wait for zoom animation to finish 
+    await bgImageLocator.hover();
+    const imageMouseZoomedIn = await page.locator(backgroundImageSelector).boundingBox();
+    expect(imageMouseZoomedIn.height).toBeGreaterThan(originalImageDimensions.height);
+    expect(imageMouseZoomedIn.width).toBeGreaterThan(originalImageDimensions.width);
+
+    // Center the mouse pointer
+    await page.mouse.move(imageCenterX, imageCenterY);
+
+    // Pan Imagery Hints
+    console.log('process.platform is '+process.platform);
+    const expectedAltText = process.platform === 'linux' ? 'Ctrl+Alt drag to pan' : 'Alt drag to pan';
+    const imageryHintsText = await page.locator('.c-imagery__hints').innerText();
+    expect(expectedAltText).toEqual(imageryHintsText);
+
+    // Click next image button
+    const nextImageButton = await page.locator('.c-nav--next');
+    await nextImageButton.click();
+
+    // Click fixed timespan button
+    await page.locator('.c-button__label >> text=Fixed Timespan').click();
+
+    // Click local clock 
+    await page.locator('.icon-clock >> text=Local Clock').click();
+
+    // Zoom in on next image
+    await bgImageLocator.hover();
+    await page.mouse.wheel(0, deltaYStep * 2);
+    const zoomedNextBoundingBox = await bgImageLocator.boundingBox();
+    const imageNextCenterX = zoomedNextBoundingBox.x + zoomedNextBoundingBox.width / 2;
+    const imageNextCenterY = zoomedNextBoundingBox.y + zoomedNextBoundingBox.height / 2;
+
+    // Verify real-time/fixed-time imagery will pause
+    const pausePlayButton = await page.locator('.c-button.pause-play');
+    expect(pausePlayButton).toHaveClass('c-button icon-pause pause-play is-paused');
+
+    // Wait for zoom animation to finish
+    await bgImageLocator.hover();
+    const imageNextMouseZoomedIn = await page.locator(backgroundImageSelector).boundingBox();
+    expect(imageNextMouseZoomedIn.height).toBeGreaterThan(originalImageDimensions.height);
+    expect(imageNextMouseZoomedIn.width).toBeGreaterThan(originalImageDimensions.width);
+
+     // Click previous image button
+     await previousImageButton.click();
+
+     // Verify previous image
+     await expect(selectedImage).toBeVisible();
+
+    // Verify real-time/fixed-time imagery will pause
+    expect(pausePlayButton).toHaveClass('c-button icon-pause pause-play is-paused');
+
+    // Wait 20ms to verify no new image has come in
+    await page.waitForTimeout(21);
+
+    // Verify next image is Disabled
+    await nextImageButton.isDisabled();
+
+    //Get background-image url from background-image css prop
+    const backgroundImage = await page.locator('.c-imagery__main-image__background-image');
```

Comment:
One alternative to this approach is there is an `img` tag that has the src more readily available. Its selector is `.c-imagery__main-image__image js-imageryView-image`.